app/models/health/careplan.rb

```diff
@@ -308,14 +308,29 @@ def active?
 
     CP2_DATE = '2023-04-01'.to_date.freeze
     def completed?
-      cp1 = provider_signed_on && patient_signed_on && [provider_signed_on, patient_signed_on].max < CP2_DATE
-      cp2 = patient_signed_on && patient_signed_on >= CP2_DATE && rn_approval?
+      # CP1 PCTPs are completed when fully signed
+      return provider_signed_on && patient_signed_on if cp1?
+      # CP2 PCTPs are completed when sent to the PCP
+      return careplan_sent_on.present? unless cp1?
```

Comment:
maybe?
```suggestion
      return careplan_sent_on.present? if cp2?
```

Otherwise you'll never hit line 316.

Reply:
Good point... I added `cp2?` after I wrote this...